src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -73,13 +72,20 @@ class RoutePathView extends React.Component<IRoutePathViewProps, IRoutePathViewS
     }
 
     private initializeAsAddingNew = async () => {
-        if (!this.props.routePathStore!.routePath) {
-            this.props.routePathStore!.setRoutePath(await this.createNewRoutePath());
-        } else {
-            this.props.routePathStore!.setRoutePath(
-                RoutePathFactory.createNewRoutePathFromOld(this.props.routePathStore!.routePath!));
+        try {
+            if (!this.props.routePathStore!.routePath) {
+                this.props.routePathStore!.setRoutePath(await this.createNewRoutePath());
+            } else {
+                this.props.routePathStore!.setRoutePath(
+                    RoutePathFactory.createNewRoutePathFromOld(
+                        this.props.routePathStore!.routePath!,
+                    ),
+                );
+            }
+            this.props.toolbarStore!.selectTool(ToolbarTool.AddNewRoutePathLink);
+        } catch (ex) {
```

Comment:
I have used catch (e) before. What ex refers to?

Reply:
Both 'e' and 'ex' refers to exception, and both are widely used globally. But could change it to 'e' to keep it consistent within our codebase.